xla/service/gpu/gpu_offloading_test.cc

```diff
@@ -113,6 +113,11 @@ class GpuCompilerTest : public HloTestBase {
   float transcendentals_per_second_{1.0f};
 };
 
+// The following test is disabled as the copy-start/copy-done IRs don't have
+// the expected host memory space colors before HLO rematerialization.
+// The issue regarding GPU layout assignment is described here:
+// https://github.com/openxla/xla/issues/10826
+#ifdef ENABLE_ON_GPU
```

Comment:
Normally DISABLED_ON_GPU is used: https://github.com/openxla/xla/blob/main/xla/tests/test_macros.h#L32

Reply:
Followed and used DISABLED_ON_GPU macro.

Reply:
OK ! Please post the comments after relevant change is submitted, otherwise the state is confusing.